docs/advanced/document-store/benchmark.md

```diff
@@ -178,17 +446,15 @@ favours the backends that use the resources efficiently. Therefore, some backend
 GRPC instead of HTTP, or use batch operations. We're okay with that, as long as it benefits the DocArray and Jina 
 user.
 * **Open to improvements**: We are constantly improving the performance of storage backends from the DocArray side and 
-updating the benchmarks accordingly. If you believe we missed an optimization (e.g. perform an operation in batches, benefit from a recent feature in upstream, avoid unnecessary steps), feel free to [raise a PR or issue](https://github.com/jina-ai/docarray/issues/new/choose). We're open to your contributions!
+updating the benchmarks accordingly. If you believe we missed an optimization (e.g. perform an operation in batches, benefit from a recent feature in upstream, avoid unnecessary steps), feel free to [raise a PR or issue](https://github.com/docarray/docarray). We're open to your contributions!
 
 ### Incompleteness on the stores
 
-We do not yet cover the following backends for various reasons:
-* **Milvus**: currently DocArray does not integrate with Milvus. We're open for contributions to DocArray's repository to 
-support it.
-* **Pinecone**: Pinecone support is coming soon. We'll add it to these benchmarks once available.
-* **Faiss, Annoy, Scann**: We do not benchmark algorithms or ANN libraries. We only benchmark backends that can be used as 
+We do not benchmark algorithms or ANN libraries like **Faiss, Annoy, Scann**. We only benchmark backends that can be used as 
 Document stores. Actually we do not benchmark HNSW itself, but it is used by some backends internally.
 
+Other storage backends that support vector search are not integrated with DocArray yet. We're open to contributions to DocArray's repository to support them.
+
```

Comment:
```suggestion
#### Client/Server setup
```
Let's say here that in our benchmarks the client and server live on the same machine to minimize network overhead.
We need to say that although a real life scenario would be clients and server living  on different machines with potentially multiple clients in parallel, we choose to keep both in the same machine and have only 1 client process